File: tests/e2e/fixtures/rust/simple.rs
// Simple Rust test fixture for debugging.

fn calculate(a: i32, b: i32) -> i32 {
    let result = a + b;  // Line 4: breakpoint target
    result
}

fn main() {
    let x = 10;
    let y = 20;
    let total = calculate(x, y);  // Line 11: breakpoint target
    println!("Result: {}", total);
}

File: tests/e2e/fixtures/javascript/simple.js
/**
 * Simple JavaScript test fixture for debugging.
 */

function calculate(a, b) {
    const result = a + b;  // Line 6: breakpoint target
    return result;
}

function main() {
    const x = 10;
    const y = 20;
    const total = calculate(x, y);  // Line 13: breakpoint target
    console.log(`Result: ${total}`);
}

main();
